zeebe/exporters/camunda-exporter/src/main/java/io/camunda/exporter/metrics/ExportDurationObserver.java

```diff
@@ -59,6 +59,7 @@ public void observeDurations() {
     // ensure we don't observe the same record write timestamp twice
     writeTimestamps.clear();
     timestamps.stream()
+        .filter(t -> t < 0) // ignore the "null" or default values
```

Comment:
❓ How can this be null or smaller zero? 🤔 This must be set right?

Reply:
AFAIK it cannot be -1, but the Agrona collection pre allocates the array and sets some null value. We fill it up, so should be fine, but I was just being defensive here.